python/demo/poisson/demo_poisson.py

```diff
@@ -212,9 +229,11 @@
     # If pyvista environment variable is set to off-screen (static)
     # plotting save png
     if pyvista.OFF_SCREEN:
+        # Run demo with 'PYVISTA_OFF_SCREEN=true python demo_poisson.py'
```

Comment:
I wonder it we want this. It's a complication introduced by CI and docker.

Reply:
I removed the saving to png part. I guess there will be a separate demo about `pyvista` where this can be explained in more detail.